README.md

```diff
@@ -30,6 +30,8 @@ A first version of the project can be found in the [chis-interoperability](https
 1. Set up a local CHT instance using theses [instructions](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/)
 1. Set up a proxy to your local CHT instance by running using something like [nginx-local-ip](https://github.com/medic/nginx-local-ip) or [ngrok](https://ngrok.com/) and update the `CHT_URL` value in the `.env` file with the new URL.
```

Comment:
> value in the `.env` file with the new URL.

I suggest adding extra step before this step and mentioning the `.env` file creation, and the existence of the `env.template` file.
Mentioning `CHT_USERNAME` & `CHT_PASSWORD` can be helpful too, as the users might use different credentials.